pivot_root: add page (#19673)
tldr-pages/tldr: pages/linux/pivot_root.md

# pivot_root

> Change the root filesystem to a new directory and move the current root to a subdirectory of the new root.

> > Commonly used during system initialization (e.g., in initramfs) to switch from a temporary root to the real root filesystem.

Reviewer: ```suggestion
> Commonly used during system initialization (e.g., in `initramfs`) to switch from a temporary root to the real root filesystem.
```

> > More information: <https://manned.org/pivot_root>.

Reviewer 2: ```suggestion
> More information: <https://manned.org/pivot_root.8>.
```

> - Move the current root to `/old_root` and make `/new_root` the new root:

Reviewer 2: ```suggestion
- Make `/new_root` the new root and move current root to a subdirectory of it:
```

> `sudo pivot_root /new_root /new_root/old_root`

Reviewer 2: Can you give me an example on how this should look like in a real world scenario?
